mobile/src/viewModels/ChooseStrategyViewModel.ts

```diff
@@ -0,0 +1,84 @@
+import { observable} from 'mobx';
+import { StrategyIded, DisplayStrategyIded, DomainIded } from '../../../mobile/src/constants/QoL';
+
+export default class ChooseStrategyViewModel {
+
+  private _selectedDomains: string[];
+  private _allStrategies: DisplayStrategyIded[];
+
+  @observable
+  public availableStrategies: DisplayStrategyIded[];
+
+  @observable
+  public selectedStrategies: StrategyIded[];
+
+  @observable
+  public learnMoreStrategy: StrategyIded;
+
+  get selectedDomains(): string[] {
+    return this._selectedDomains;
+  }
+
+  constructor() {
+
+      this._selectedDomains, this._allStrategies, this.availableStrategies, this.selectedStrategies = [];
+      this.learnMoreStrategy = {
+        id: '',
+        title: '',
+        details: '',
+        slugs: [''],
+      }
```

Comment:
this feels janky... Can we add some sort of type that encapsulates Strategy + LearnMore

Reply:
changed to init to null and checking for null before moving to details view